### ISSUE TITLE
Some Foreign Keys are completely ignored casing INSERTs to fail

### ISSUE THREAD

**Issue by chaghi** [KorzhCom/EasyData#50]
This is a follow up to #49, although I'm not sure if the issues are *technically* related, other than having been discovered in the same scenario (trying to manage the ASP.NET Core Identity data model with EasyData).

EasyData seems to ignore some FK columns in some entities. The corresponding column/value is not shown in the data grid, the field is not rendered at all in the CRUD form, and any INSERT will fail because the FK ends up being NULL.

Two examples:

* `IdentityRoleClaim`: EasyData shows `Id`, `ClaimType` and `ClaimValue`. The property/column `RoleId` is completely ignored;
* `IdentityUserClaim`: Same, we have `Id`, `ClaimType` and `ClaimValue`, and `UserId` is ignored;

Trying to create a new record on either table will fail with (example for `IdentityUserClaim`):

> **Ooops, smth went wrong**
>
> Cannot insert the value NULL into column 'UserId', table 'EasyDataDB.dbo.AspNetUserClaims'; column does not allow nulls. INSERT fails. The statement has been terminated.

## Steps to reproduce

* Create a new project with "Individual accounts" (`dotnet new webapp --auth Individual`)
* Setup EasyData
* Browse for example `/easydata/IdentityRoleClaim` or `/easydata/IdentityUserClaim`

## Expected result

* An empty grid (as there are no records yet), including the column `RoleId` (for Identity Role Claim) and `UserId` (for Identity User Claim)
* Trying to add a new record should render the field `RoleId` / `UserId`, accordingly

## Actual result

* The columns `RoleId` or `UserId` are not shown in the query
* The fields `RoleId` or `UserId` are not rendered in the CRUD form

**Comment by antifree** [comment]
Hello,

Have you tried to use such option: 


```c#
options.UseDbContext<AppDbContext>(opts => opts.SkipForeignKeys = false)
```

**Comment by chaghi** [comment]
Hi, no, I wasn't aware of that option, sorry.

Anyway, I've just tried that. With `SkipForeignKeys` in `false`, the FK properties are shown as columns in the query, but they're still ommitted in the CRUD form (so is still not possible to insert a new record, or change the FK through EasyData).

**Comment by antifree** [comment]
Hello, sorry I was a little busy these days.

I hope this commitment (8c28ef8cc05f3b438e0cdfd164ad507d539e54be) will fix this problem

**Comment by chaghi** [comment]
Hi, no problem!

I've just tried again on an updated `dev` branch, but unless I'm doing something wrong (for some reason webpack is not called automatically when building, so I'm calling it manually (`webpack --mode=development`)), the CRUD forms are not fixed. For example, IdentityRoleClaims still fails to render the RoleId field:

![image](https://user-images.githubusercontent.com/6282941/128704189-3047435c-b19e-4f29-aae1-0db959aa1969.png)

**Comment by antifree** [comment]
To update to the latest changes, you have to run in `easydata.js` folder
```
npm run build:prod 
```

And then run watch in the sample with identity (if you use ours (`EasyDataAspNetCoreTest02`))

Here is a screenshot of Create Identity Role Claim form on my side
![image](https://user-images.githubusercontent.com/28184848/128759754-625b0537-2797-4367-8e5f-93a649327429.png)

P.s. Be sure, that you have also set this option: 
```c#
options.UseDbContext<AppDbContext>(opts => opts.SkipForeignKeys = false)
```

**Comment by chaghi** [comment]
Hi, thanks for taking the time to explain that, I appreciate it!
I can confirm that having rebuilt the frontend as per your instructions, the issue appears to be fixed.

**Comment by antifree** [comment]
Cool!

Thank you for testing. 

I hope we will publish the update tomorrow.

**Comment by antifree** [comment]
Fixed in version 1.3.2. PR #51
